dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZKServer.java

```diff
@@ -94,8 +112,9 @@ public void shutdown() {
      *
      * @param port The port to listen on
      */
-    public static void startLocalZkServer(final int port) {
-        String zkDataDir = System.getProperty("user.dir") +"/zookeeper_data";
+    public void startLocalZkServer(final int port) {
+        String zkDataDir = System.getProperty("user.dir") + ("".equals(prefix) ? prefix : ("/" + prefix)) + "/zookeeper_data";
```

Comment:
```suggestion
        String zkDataDir = System.getProperty("user.dir") + (StringUtils.isBlank(prefix) ? StringUtils.EMPTY : ("/" + prefix)) + "/zookeeper_data";
```
Hi,
Good job.
Is it better to use this utils of `org.apache.dolphinscheduler.common.utils.StringUtils`?

Reply:
solved